### ISSUE TITLE
Database fields/rows shown as duplettes (structure and data view)

### ISSUE THREAD

**Issue by mbraeutigam** [sqlitebrowser/sqlitebrowser#2269]
Details for the issue
--------------------

#### What did you do?

Run

```SQL
CREATE TABLE DB000255 (VNK TEXT CONSTRAINT DB000255_VNK CHECK(typeof(VNK)='text' and length(VNK)<=10 or typeof(VNK)='null'),NNK TEXT CONSTRAINT DB000255_NNK CHECK(typeof(NNK)='text' and length(NNK)<=10 or typeof(NNK)='null'),SORT INTEGER CONSTRAINT DB000255_SORT CHECK(typeof(SORT)='integer' and SORT>=0 or typeof(SORT)='null'),XKOORD REAL CONSTRAINT DB000255_XKOORD CHECK(typeof(XKOORD)='real' or typeof(XKOORD)='integer' or typeof(XKOORD)='numeric' or typeof(XKOORD)='null'),YKOORD REAL CONSTRAINT DB000255_YKOORD CHECK(typeof(YKOORD)='real' or typeof(YKOORD)='integer' or typeof(YKOORD)='numeric' or typeof(YKOORD)='null'),STAND DATE CONSTRAINT DB000255_STAND CHECK(typeof(STAND)='text' and datetime(STAND) IS NOT NULL or typeof(STAND)='null'),FLAG TEXT)
```

Open database structure view || data view.

#### What did you expect to see?
database structure
![image](https://user-images.githubusercontent.com/4228180/83858452-d2749500-a71c-11ea-904b-81404ebe2301.png)

data view
![image](https://user-images.githubusercontent.com/4228180/83858750-3eef9400-a71d-11ea-81fd-8b1c5bd39f80.png)

#### What did you see instead?
database structure
![image](https://user-images.githubusercontent.com/4228180/83858427-c4bf0f80-a71c-11ea-96ba-4354191226fb.png)

data view
![image](https://user-images.githubusercontent.com/4228180/83858715-3303d200-a71d-11ea-95ce-8a552d12f356.png)

Useful extra information
-------------------------

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?

- [X] Windows: ( _version:_ 10 )
- [ ] Linux: ( _distro:_ ___ )
- [ ] macOS: ( _version:_ ___ )
- [ ] Other: ___

#### What is your DB4S version?

- [X] 3.12.0-(alpha or betas) NIGHTLY
- [ ] 3.11.2
- [ ] 3.11.1
- [ ] 3.10.1
- [ ] Other: ___

#### Did you also

- [X] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [X] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by MKleusberg** [comment]
Thanks, looks like we need to figure out this issue before the next release. I'll look into it in a second.

**Comment by MKleusberg** [comment]
Looks like I introduced this problem in 139de6690a8e9220d5a9a51dd69251016e6bd278. So if we don't cherry pick this commit, we don't need to cherry pick the fix for this one either.

Anyway, the problem is caused by the named constraint (e.g. the ```DB000255_VNK``` in ```VNK TEXT CONSTRAINT DB000255_VNK CHECK(...)```). Because we can't store this name yet (planning to implement that for the 3.13 release!), we mark the table as not fully parsed. You might notice the warning because of that when trying to modify the table. With the commit I mentioned above I added a fall back which asks SQLite itself for some information in this case but unfortunately it was too sensitive. Now it's only asking SQLite if we could not figure out anything about the table.

In short, it should be fixed in tomorrow's nightly build. Can you double check if it's working for you with that?

**Comment by mbraeutigam** [comment]
Sure, will get back to you tomorrow.

**Comment by mbraeutigam** [comment]
OK - seems fine now.

![image](https://user-images.githubusercontent.com/4228180/83942971-b1817200-a7f8-11ea-9adb-64c6350ef96e.png)

**Comment by MKleusberg** [comment]
Awesome! Thank you again for reporting this :smile:
